roles/nvidia_smi/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: check for nvidia_smi
+  stat:
+    path: '{{ nvidia_smi_path }}nvidia-smi'
+  register: nvidia_smi_path
+  ignore_errors: yes
+
+
+- block:
+  - name: capture nvidia_smi info
+    command: nvidia-smi -q -x
```

Comment:
Can we also pass the full path? `{{ nvidia_smi_path }}nvidia-smi` ?

Reply:
Ok, noted and fixing. To confirm, you mean putting all of "/usr/bin/nvidia-smi" inside of the var right?

Reply:
Right. Unless there are other nvidia tools that we will be calling, we should just store the binary path in a var.

Reply:
no other nvidia tools will be called from this role. Will fix that and push soon. Thanks.